pkg/kubectl/describe.go

```diff
@@ -173,13 +173,24 @@ func (d *NamespaceDescriber) Describe(namespace, name string, describerSettings
 	}
 	resourceQuotaList, err := d.Core().ResourceQuotas(name).List(api.ListOptions{})
 	if err != nil {
-		return "", err
+		if errors.IsNotFound(err) {
+			// Server does not support resource quotas.
+			// Not an error, will not show resource quotas information.
+			resourceQuotaList = nil
+		} else {
+			return "", err
```

Comment:
Yes thats a good point. However doing that will require some refactoring.
I will need to update describeNamespace to take errors for both resourceQuota and limit ranges and then update describeResourceQuota and describeLimitRanges to take errors as well.
Or I could directly pass the output of describeLimitRanges and describeResourceQuotas to describeNamespace.
But I decided not to make those changes in this PR, since that case is not critical for us.

Let me know if you want me to include that in this PR.